src/nls/fi/strings.js

```diff
@@ -154,6 +154,7 @@ define({
     "FIND_QUERY_PLACEHOLDER"            : "Etsi\u2026",
     "REPLACE_PLACEHOLDER"               : "Korvaa merkkijonolla\u2026",
     "BUTTON_REPLACE_ALL"                : "Kaikki\u2026",
+    "BUTTON_REPLACE_BATCH"              : "Sarja\u2026",
```

Comment:
Not sure about this. Maybe something like "Korjaa erissä" or something, "Erä" sounds kinda weird, "sarja" too.